core/domain/user_jobs_one_off.py

```diff
@@ -324,3 +326,54 @@ def map(user_model):
     @staticmethod
     def reduce(key, stringified_values):
         pass
+
+
+class UserRolesMigrationOneOffJob(jobs.BaseMapReduceJobManager):
+    """A one time job to attach a new field (representing role of user) to the
+    UserSettingsModel. The job will load all existing users from the datastore,
+    look for the role to assign them and then store them back thus updating the
+    table schema and adding appropiate roles to users.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [user_models.UserSettingsModel]
+
+    @staticmethod
+    def map(user_model):
+        admin_usernames = config_domain.Registry.get_config_property(
```

Comment:
I'm not sure if there's a way to just get these once, instead of getting them for for every user. @seanlip -- do you have any idea?